tests/testthat/test-DFFilterStates.R

```diff
@@ -99,38 +99,38 @@ testthat::test_that("DFFilterStates$set_filter_state updates filter state which
 })
 
 testthat::test_that("DFFilterStates$set_filter_state throws error when using an unnamed list",
-  code = {
-    dffs <- DFFilterStates$new(
-      input_dataname = "iris",
-      output_dataname = "iris_filtered",
-      datalabel = character(0),
-      varlabels = character(0),
-      keys = character(0)
-    )
-    fs <- list(
-      c(5.1, 6.4),
-      Species = c("setosa", "versicolor")
-    )
-    testthat::expect_error(dffs$set_filter_state(state = fs, data = iris))
-  }
+                    code = {
```

Comment:
Please make this indent smaller
```suggestion
testthat::test_that("DFFilterStates$set_filter_state throws error when using an unnamed list", {
```